File: Complete/controllers/profileTester.py
import iCUE
from os import listdir
from os.path import isfile, join

game = input('Game Name: ')
iCUE.SetGame(game)

while True:
	state = input('State Name: ')
	iCUE.SetState(game, state)
	input('Enter to clear...')
	iCUE.ClearState(game, state)

File: Example Controllers/Ruby/profileTester.rb
require_relative './iCUE'

print "Game Name: "
$game_name = gets.chomp
ICUE.set_game($game_name)

while (true)
	print "State Name: "
	state_name = gets.chomp
	ICUE.set_state($game_name, state_name)
	puts "Enter to clear..."
	gets
	ICUE.clear_state($game_name, state_name)
end